go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/ad-freiburg/wharfer
 
 require github.com/docker/docker v1.13.1
+
+go 1.13
```

Comment:
Is go 1.13 specific code used? It compiles fine with go1.12.7 on Fedora, that's why I'm asking.

Reply:
This was done automatically by `go mod tidy`